Dockerfile.spark

```diff
@@ -12,7 +12,11 @@ RUN apt-get update && apt-get install -yq --no-install-recommends --force-yes \
     rm -rf /var/lib/apt/lists/*
 
 RUN easy_install3 pip py4j
```

Comment:
Remove `pip` from this line. You install it below

Reply:
See the response I posted below. Thanks!